3_Regression/Simple_Linear_Regression/python/python-scratch/LinearRegression_scratch.py

```diff
@@ -0,0 +1,118 @@
+# Linear Regression Model.
```

Comment:
You are clearly contributing someone else's code?

Reply:
Sir it's mine, i'm 2nd year student and my domain is AIML.
Specifically, it's one of my assignment from 2nd semester.